### DISCUSSION TITLE
Let's Know this Repo

### DISCUSSION THREAD

**Discussion by Niketkumardheeryan** [Niketkumardheeryan/ML-CaPsule#243] [General]
Hey,

**Comment by Sushila-beniwal** [comment]
@Niketkumardheeryan
Hello sir, the issue which are already assigned to the contributor can other contributor also ask to work on the same issue?

**Comment by Mahatva1303** [comment]
Yes, you can still ask to work on the issue, but it usually depends on the project maintainer’s decision. Some maintainers allow multiple contributors to collaborate on the same issue, while others prefer keeping it assigned to one person to avoid duplicate work.
If the assigned contributor is inactive for a long time, you can politely ask the maintainer whether you may work on it instead.

**Reply by Mahatva1303** [reply]
@Sushila-beniwal hope this answers your question if yes please mark it as answered
